rules/react/walmart.js

```diff
@@ -19,16 +19,28 @@ module.exports = {
     "react/jsx-closing-bracket-location": [2, 'tag-aligned'],
     // Enforce or disallow spaces inside of curly braces in JSX attributes
     "react/jsx-curly-spacing": 0,
+    // Enforce event handler naming conventions in JSX
+    "react/jsx-handler-names": 0,
     // Validate props indentation in JSX
     "react/jsx-indent-props": [2, 2],
+    // Validate JSX has key prop when in array or iterator
+    "react/jsx-key": 0,
     // Limit maximum of props on a single line in JSX
     "react/jsx-max-props-per-line": 0,
+    // Prevent usage of .bind() and arrow functions in JSX props
+    "react/jsx-no-bind": 0,
```

Comment:
I didn't know this was a problem :scream: but it totally makes sense. On the one hand this could cause some serious GC thrashing but on the other it takes away the ability to use simple arrow functions :/